<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>activatoncodeclass</class>
 <widget class="QDialog" name="activatoncodeclass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/ICON/photo3(1).png);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1311</width>
     <height>745</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>381</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
border: 3px dashed rgb(85, 170, 127);
margin-top: 1ex; /* leave space at the top for the title */
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;
}

QGroupBox::title {
subcontrol-origin: margin;
padding: 0 3px;

}
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="code">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #E1E1E1, stop: 0.4 #DDDDDD,
        stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 1px solid gray;
    border-radius: 5px;
    padding: 1px 18px 1px 3px;
}
QLineEdit::down-arrow {
    width: 0;
    height: 0;
    border-style: solid;
    border-width: 6px 6px 0 6px;
    border-color: gray transparent transparent transparent;
}
QLineEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: gray;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
    background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 #f0f0f0, stop: 1 #d8d8d8);

}
QLineEdit::hover {
    background: #d0d0d0;
}

QLineEdit::pressed {
    background: #c8c8c8;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="loginemail_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;}
</string>
      </property>
      <property name="text">
       <string>Entrer Code</string>
      </property>
     </widget>
     <widget class="QPushButton" name="verififercode">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>270</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background: rgb(66, 117, 102);
border-radius:10px;
border-style:solid;
border-width:2px;
border-color:#9999aa;
color:white;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;
}
QPushButton:enabled{
background-color: rgb(66, 117, 102);
color:white;
}

QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:0.5, y1:0.0342727, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
color:#fffffe;
}

QPushButton:hover:!pressed{
background-color: rgb(221,222,217);
color:#0c2f70;
}

QPushButton:disabled{
background-color:rgb(66, 117, 102);
color:#ffffff;
}
</string>
      </property>
      <property name="text">
       <string>Verifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
qproperty-alignment:AlignCenter;
color:rgb(66, 117, 102);

}</string>
      </property>
      <property name="text">
       <string>Oublier Mot de pass</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>401</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
border: 3px dashed rgb(85, 170, 127);
margin-top: 1ex; /* leave space at the top for the title */
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;
}

QGroupBox::title {
subcontrol-origin: margin;
padding: 0 3px;

}
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>290</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background: rgb(66, 117, 102);
border-radius:10px;
border-style:solid;
border-width:2px;
border-color:#9999aa;
color:white;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;
}
QPushButton:enabled{
background-color: rgb(66, 117, 102);
color:white;
}

QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:0.5, y1:0.0342727, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
color:#fffffe;
}

QPushButton:hover:!pressed{
background-color: rgb(221,222,217);
color:#0c2f70;
}

QPushButton:disabled{
background-color:rgb(66, 117, 102);
color:#ffffff;
}
</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
qproperty-alignment:AlignCenter;
color:rgb(66, 117, 102);

}</string>
      </property>
      <property name="text">
       <string>Entrer nouveau mot de pass</string>
      </property>
     </widget>
     <widget class="QLabel" name="loginemail">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;}
</string>
      </property>
      <property name="text">
       <string>Nouveau Mot de pass</string>
      </property>
     </widget>
     <widget class="QLabel" name="loginmdp">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;}
</string>
      </property>
      <property name="text">
       <string>Confirmer Mot de pass</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="confmdp">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #E1E1E1, stop: 0.4 #DDDDDD,
        stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 1px solid gray;
    border-radius: 5px;
    padding: 1px 18px 1px 3px;
}
QLineEdit::down-arrow {
    width: 0;
    height: 0;
    border-style: solid;
    border-width: 6px 6px 0 6px;
    border-color: gray transparent transparent transparent;
}
QLineEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: gray;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
    background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 #f0f0f0, stop: 1 #d8d8d8);

}
QLineEdit::hover {
    background: #d0d0d0;
}

QLineEdit::pressed {
    background: #c8c8c8;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="mdp">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #E1E1E1, stop: 0.4 #DDDDDD,
        stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 1px solid gray;
    border-radius: 5px;
    padding: 1px 18px 1px 3px;
}
QLineEdit::down-arrow {
    width: 0;
    height: 0;
    border-style: solid;
    border-width: 6px 6px 0 6px;
    border-color: gray transparent transparent transparent;
}
QLineEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: gray;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
    background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 #f0f0f0, stop: 1 #d8d8d8);

}
QLineEdit::hover {
    background: #d0d0d0;
}

QLineEdit::pressed {
    background: #c8c8c8;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>175</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Chiffres</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>175</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>minuscule</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>175</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Majuscule</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom_2">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>460</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background: rgb(221,222,217);
    border: 0px ;
    border-radius: 0px;
  
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::NoEcho</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
